### PR TITLE
Fix #1712: Liquibase 1.9.x fails on Oracle

### LINKED ISSUE #1712: Liquibase 1.9.x fails on Oracle
```[2024-10-03 12:09:19] SEVERE [liquibase.integration] Migration failed for changeset powerauth-java-server/1.9.x/20240718-add-temporary-keys.xml::1::Petr Dvorak:
app-1      |      Reason: liquibase.exception.DatabaseException: ORA-02270: no matching unique or primary key for this column-list
app-1      |  [Failed SQL: (2270) CREATE TABLE POWERAUTH.pa_temporary_key (id VARCHAR2(37) NOT NULL, application_key VARCHAR2(32) NOT NULL, activation_id VARCHAR2(37), private_key_encryption INTEGER DEFAULT 0 NOT NULL, private_key_base64 VARCHAR2(255) NOT NULL, public_key_base64 VARCHAR2(255) NOT NULL, timestamp_expires TIMESTAMP NOT NULL, CONSTRAINT PK_PA_TEMPORARY_KEY PRIMARY KEY (id), CONSTRAINT pa_temporary_activation_id_fk FOREIGN KEY (activation_id) REFERENCES POWERAUTH.pa_activation(activation_id), CONSTRAINT pa_temporary_key_application_key_fk FOREIGN KEY (application_key) REFERENCES POWERAUTH.pa_application_version(application_key))]```

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

**Review by romanstrobl — DISMISSED** [review]
Looks OK.

File: docs/db/changelog/changesets/powerauth-java-server/1.9.x/20241004-add-unique-constrains.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  ~
+  ~ You should have received a copy of the GNU Affero General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" logicalFilePath="powerauth-java-server/1.9.x/20241004-add-unique-constrains.xml" author="Lubos Racansky">
```
Comment (zcgandcomp) [inline docs/db/changelog/changesets/powerauth-java-server/1.9.x/20241004-add-unique-constrains.xml:24]: nedávat UK na PK

**Review by zcgandcomp — CHANGES_REQUESTED** [review]
There is no need to add UK on existing PK - IMHO this will fail.

**Review by zcgandcomp — APPROVED** [review]
Looks OK

**Pull request merged**